tools/Customization/DevHome.Customization/Views/FileExplorerPage.xaml.cs

```diff
@@ -108,4 +109,14 @@ public void RegisterRepository(IExtensionWrapper localRepositoryProvider, string
 
         ViewModel.RefreshTrackedRepositories();
     }
+
+    private void RemoveButton_Click(object sender, RoutedEventArgs e)
+    {
+        if (ValidateRepositoryPath(RemovePathTextBox.Text))
```

Comment:
I'm not sure validation is a good idea for removal. What if the folder has been deleted, or exists but no longer contains a repository? I think if we want to unregister a repository folder, we should just go ahead and do it.

Reply:
+1.  Removing things is best-effort.

Reply:
Addressed in latest. Thanks!